frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -185,6 +188,7 @@ class EditEmail extends Component {
                     value={EMAIL_TYPE.reply}
                     checked={replyChecked}
                     className="visually-hidden"
+                    disabled={isDisabled}
                   />
```

Comment:
Instead of creating a const, can you not just do the following?
`disabled={this.props.disabled}`

Reply:
Thought it would not work if I did it that way, but it does :P changed in e89df6c